frontend/public/components/dashboard/dashboards-page/cluster-dashboard/cluster-dashboard.tsx

```diff
@@ -14,29 +14,28 @@ import { useK8sGet } from '../../../utils/k8s-get-hook';
 import { InfrastructureModel } from '../../../../models';
 import { K8sResourceKind } from '../../../../module/k8s';
 import { ClusterDashboardContext } from './context';
+import { useUserSettings } from '@console/shared';
 
 const mainCards = [{ Card: StatusCard }, { Card: UtilizationCard }];
 const leftCards = [{ Card: DetailsCard }, { Card: InventoryCard }];
 const rightCards = [{ Card: ActivityCard }];
 
+const HIDE_QUICK_START_DASHBOARD_TILE_CONFIG_MAP_KEY = 'console.dashboard.quickStartTile';
+
 export const ClusterDashboard: React.FC<{}> = () => {
-  const isQuickStartsCatalogCardHidden =
-    localStorage.getItem(HIDE_QUICK_START_DASHBOARD_TILE_STORAGE_KEY) === 'true';
   const [infrastructure, infrastructureLoaded, infrastructureError] = useK8sGet<K8sResourceKind>(
     InfrastructureModel,
     'cluster',
   );
-  const [showQuickStartsCatalogCard, setShowQuickStartsCatalogCard] = React.useState<boolean>(
-    !isQuickStartsCatalogCardHidden,
+  // [TODO](sahil143): use sync capability here
```

Comment:
will this be dependent on https://github.com/openshift/console/pull/7298 ?

Reply:
No, it doesn't have to as it only requires adding a third param which can be handled seperatly.